### PR TITLE
Feature/uai 2018

### PR DESCRIPTION
**Pull request by mspector**

### CODE REVIEW

File: venues/auai.org/UAI/2018/webfield/recruitSPCWebfield.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+
+// ------------------------------------
+// Basic venue homepage template
```
Comment (zbialecki) [inline venues/auai.org/UAI/2018/webfield/recruitSPCWebfield.js]: Can you update the comments?

File: venues/auai.org/UAI/2018/webfield/recruitSPCWebfield.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+// Constants
+var CONFERENCE_ID = 'auai.org/UAI/2018';
+
+// Main is the entry point to the webfield code and runs everything
+function main() {
+  Webfield.ui.setup('#invitation-container', CONFERENCE_ID);  // required
+
+  renderConferenceHeader();
+
+  // Webfield.get('/notes', {id: args.noteId}).then(render);
```
Comment (zbialecki) [inline venues/auai.org/UAI/2018/webfield/recruitSPCWebfield.js]: And maybe remove this one

File: venues/auai.org/UAI/2018/process/recruitSPCProcess.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  var or3client = lib.or3client;
+  var hashKey = or3client.createHash(note.content.email, "2810398440804348173");
+  if(hashKey == note.content.key) {
+    if (note.content.response == 'Yes') {
+      console.log("Invitation replied Yes")
+      //if a user is in the declined group, remove them from that group and add them to the reviewers group
+      or3client.removeGroupMember('auai.org/UAI/2018/Senior_Program_Committee/Declined', note.content.email, token)
+      or3client.addGroupMember('auai.org/UAI/2018/Senior_Program_Committee', note.content.email, token)
+      .then(result => done())
+      .catch(error => console.log(error));
```
Comment (melisabok) [inline venues/auai.org/UAI/2018/process/recruitSPCProcess.js]: you need to call the done function

File: venues/auai.org/UAI/2018/process/recruitSPCProcess.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    } else if (note.content.response == 'No'){
+      console.log("Invitation replied No")
+      //if a user is in the reviewers group, remove them from that group and add them to the reviewers-declined group
+      or3client.removeGroupMember('auai.org/UAI/2018/Senior_Program_Committee', note.content.email, token)
+      or3client.addGroupMember('auai.org/UAI/2018/Senior_Program_Committee/Declined', note.content.email, token)
+      .then(result => done())
+      .catch(error => done(error));
+    }
+    return true;
```
Comment (melisabok) [inline venues/auai.org/UAI/2018/process/recruitSPCProcess.js]: same here.

Comment (mspector) [reply venues/auai.org/UAI/2018/process/recruitSPCProcess.js]: even though it's being called in both line 18 and 19? where else does it need to go?

Comment (zbialecki) [reply venues/auai.org/UAI/2018/process/recruitSPCProcess.js]: When note.content.response isn't 'Yes' or 'No'

**Pull request merged**